README.md

```diff
@@ -44,6 +44,34 @@ where `extras` can be one or more of `ipex`, `neural-compressor`, `openvino`, `n
 
 # Quick tour
 
+## IPEX
+Below are examples of how to use IPEX model to generate texts.
+### generate
+```diff
+import torch
+from transformers import AutoTokenizer, AutoConfig
+- from transformers import AutoModelForCausalLM
++ from optimum.intel.ipex import IPEXModelForCausalLM
+
+config = AutoConfig.from_pretrained("gpt2")
+model = IPEXModelForCausalLM.from_pretrained(
+  "gpt2",
```

Comment:
do you have a quantized model under the intel organization https://huggingface.co/Intel that we can directly load instead ?

Reply:
I am afraid not, but I could upload a quantized one. 
Inputting a common model_id will be more acceptable for users because they might want to try different models with the API. Would like to hear your opinion : )

Reply:
I'm not sure I agree here, it would be nice to load an IPEX quantized model here to promote this feature support. If we don't do it we should then explain the gain in using IPEX vs torch : what optimization are applied depending on whether we epxort the model to torchscript or not.

Also could you move this section after the OpenVINO section so that we keep the same structure ?